src/EditorFeatures/VisualBasic/NavigationBar/VisualBasicNavigationBarItemService.vb

```diff
@@ -65,23 +67,23 @@ Namespace Microsoft.CodeAnalysis.Editor.VisualBasic.NavigationBar
             Return TypeOf item IsNot AbstractGenerateCodeItem
         End Function
 
-        Private Function GetTypesInFile(semanticModel As SemanticModel, cancellationToken As CancellationToken) As IEnumerable(Of INamedTypeSymbol)
+        Private Function GetTypesAndPositionsInFile(semanticModel As SemanticModel, cancellationToken As CancellationToken) As IEnumerable(Of Tuple(Of INamedTypeSymbol, Integer))
```

Comment:
I'm never a fan of "magic integers" in tuples...you could pass the syntax node directly if that bugs you too.